### ISSUE TITLE
Support for elmish with commands

### ISSUE THREAD

**Issue by OnurGumus** [Shmew/Feliz.Recoil#8]
Currently, I am seeing elmish is based on a simple elmish loop without commands. It would be nice if full elmish with commands are supported.

**Comment by Shmew** [comment]
Yeah currently it's not elmish as much as the traditional React reducer pattern. I originally had full elmish support, but it's not quite compatible with Recoil and can lead to race condition issues. This is especially the case when React concurrent mode is released, which Recoil will be compatible.

They're currently working on a way to support effects within the context of Recoil that I'm hoping will solve this problem.

**Comment by Shmew** [comment]
AtomEffects are released now, but I'll need to give it some thought as to how (if) this can be done. In the mean time [this](https://shmew.github.io/Feliz.Recoil/#/Recoil/Examples/Effects) might help.
